content/stability-working-group/index.md

```diff
@@ -0,0 +1,73 @@
+
+But we just said that change is going to happen! As such, _ecosystem stability_ is managing the *impact* of change as felt by all members described above. In order to reduce the negative impacts of change to the ecosystem there must be a graceful handling path, not a lack of change entirely. For example, deprecating something before removal. Another important factor here is the timing. Being predictable in change management means that the various groups can schedule work rather than being interrupted. This is why it is critical to be able to handle change as gracefully as possible.
+
+## The Stability Working Group
+
+Who is the SWG? Membership and meeting attendance is fluid, but we have representation from a cross section of the ecosystem. That includes people involved in or with background in language design, compiler, tooling and library maintenance, industrial usage, academia and consulting.
+
+Given that, we promote tools, procedures, and general culture to effectively manage change. In order to manage this we, as a community must be able to identify when change happens, plan for upcoming changes, and minimize the breadth and depth of the impact of any given change. Finally we aim to bring representation for the sections of the ecosystem that are generally less present.
+
+## What We Have Been Working On
+
+The SWG has worked on several projects, some of which continue to this day.
+
+We worked on a [review](https://jappie.me/analyzing-haskell-stability.html) of breakages tracked by the GHC team. This was illuminating to highlight and enumerate a high level set of changes. There is still room to do more extensive analysis here as well.
+
+Another project we have been working on is defining a framework for classifying GHC language extensions. This has been a long process. But ultimately is the type of communication enhancing improvement that is needed for predictability.
+
+More concretely we have defined a set of compiler features to help with stability. These were ideas that already existed in some form. Examples that we were able to bring to attention and have since landed in GHC include warning categories and deprecation of exports. Those both allow for more communication from compiler and library authors to users.
+
+Finally, we find ourselves with a recurrent item. That is to be generally facilitators of communication. This means bringing groups together, highlighting issues, and generally fostering discussion.
+
+
+## How you can help
+
+We have been working on many things, but we can still use help! Hopefully I have intrigued, inspired, or just otherwise cajoled you into a desire to contribute. There are several ways to help. The lowest effort is to contribute to Haskell Foundation. Another is to reach out! As I said above we have several projects that could use someone to champion. The SWG often hosts guests at meetings to discuss particular issues or concerns. You do not have to commit ongoing time in order to contribute!
```

Comment:
"contribute to Haskell Foundation" - do you mean to donate?

"several projects" - it would gather more attention if you list a few specific examples (potentially repeating yourself). Re-reading the previous section it's not immediately clear for me what is ongoing and what is finished.